README.md

```diff
@@ -28,38 +28,42 @@ Example configuration for **Metadefender Cloud** usage:
 [general]
 BaseUrl=https://api.metadefender.com/v4/file
 ConstantUsers=5
-TestDuration=10
+InjectDuration=30
+MaxDuration=60
 ScanWorkflow=multiscan,sanitize
 LocalPath=/home/opswatuser/testfiles
 PollingIntervals=500
 ApiKey=1234567890abcdefghijklmnopqrstuv
-WaitBeforePolling=1000
+ScanNumberUpperBound=13
 ```
 
 Example configuration for **local MetaDefender Core** usage:
 ```
 [general]
 BaseUrl=http://localhost:8008/file
 ConstantUsers=5
-TestDuration=10
+InjectDuration=30
 LocalPath=/home/opswatuser/testfiles
 PollingIntervals=500
-WaitBeforePolling=1000
+ScanNumberUpperBound=13
 ```
 
 ***BaseURL:***
 
 MetaDefender REST URL, e.g.: https://api.metadefender.com/v4/file (Cloud) or http://localhost:8008/file (local). 
 More information about file scanning: [File scanning API](https://onlinehelp.opswat.com/mdcloud/2.1_Scanning_a_file_by_file_upload.html)
 
-***ConstantUsers:***
+***UsersPerSec:***
 
-The number of constant concurrent users for the test. Each simulated user will submit a randomly 
-selected file for scanning, wait for the scan result, then select a new file for scanning.
+The number of users injected every second for the test. Each simulated user will submit a randomly selected file for scanning and wait for the scan result. It is equal to scan request per second.
```

Comment:
It is equal to the number of scan requests per second.

Reply:
Maybe we can change the order of sentences:

The number of users injected every second during the test (i.e. the number of scan requests per second). Each simulated user will submit a randomly selected file for scanning and wait for the scan result.